workflows/iuf/operations/vcs-update/vcs-update-working-branch.yaml

```
@@ -71,6 +71,11 @@ spec:
                   value: "{{steps.get-vcs-secrets.outputs.parameters.secret_name}}"
                 - name: cf_update_gitea_org
                   value: cray #default
+                - name: cf_update_gitea_repo
+                  value: >
+                    "repo_name" in jsonpath(inputs.parameters.global_params, "$.product_manifest.current_product.manifest.vcs") ?
+                    jsonpath(inputs.parameters.global_params, "$.product_manifest.current_product.manifest.vcs.repo_name") :
+                    ""
```

Oh, I think we also need the curly braces, based on what we tested with mine. I also put the empty string on the same line as the colon, but I don't think that matters. This should work:

```suggestion
                    {{="repo_name" in jsonpath(inputs.parameters.global_params, "$.product_manifest.current_product.manifest.content.vcs") ?
                    jsonpath(inputs.parameters.global_params, "$.product_manifest.current_product.manifest.content.vcs.repo_name") :
                    ""}}
```